.mise.toml

```
@@ -0,0 +1,3 @@
+[tools]
+swiftlint = "0.54.0"
+swiftformat = "0.54.3"
```

Why not latest versions?
```suggestion
swiftlint = "0.56.1"
swiftformat = "0.54.3"
```

---

No particular reason :). I'll upgrade.